.github/workflows/single-redis-oldest.yml

```
@@ -43,7 +43,11 @@ jobs:
           path: ~/redis
           key: redis-${{ hashFiles('.download-redis.sh') }}
       - name: Setup environment
+<<<<<<< HEAD
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)